api/Console.json

```diff
@@ -0,0 +1,1048 @@
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "9",
+              "notes": [
+                "Available in workers from 38 onwards"
+              ]
+            },
+            "firefox_android": {
+              "version_added": "9",
+              "notes": [
+                "Available in workers from 38 onwards"
+              ]
+            },
+            "ie": {
+              "version_added": "11"
+            },
+            "opera": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": "4"
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "info": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/console/info",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "4",
+              "notes": [
+                "Available in workers from 38 onwards",
+                "Subtitution string available from 9 onwards"
+              ]
+            },
+            "firefox_android": {
+              "version_added": "4",
+              "notes": [
+                "Available in workers from 38 onwards",
+                "Subtitution string available from 9 onwards"
+              ]
+            },
+            "ie": {
+              "version_added": "8"
+            },
+            "opera": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": true,
+              "notes": [
+                "No information icon"
+              ]
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "log": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/console/log",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": true,
+              "notes": [
+                "Substitution strings available from 28 onwards",
```

Comment:
Same thing for the substitution strings here. Please add a sub feature and add the second note there.